### ISSUE TITLE
Once supper

### ISSUE THREAD

**Issue by boardmain** [Android-Embed/ti-firebase#2]
hi how can retrive changes once?
you support once call?

**Comment by mlabieniec** [comment]
@boardmain Yes you can use "singleValueListener" instead of valueListener same as here:
https://github.com/Android-Embed/ti-firebase/blob/master/example/lib/firebase.js#L43
